python/cudf/cudf/tests/test_parquet.py

```diff
@@ -210,6 +210,35 @@ def test_parquet_reader_basic(parquet_file, columns, engine):
     assert_eq(expect, got, check_categorical=False)
 
 
+@pytest.mark.filterwarnings("ignore:Using CPU")
+@pytest.mark.parametrize("engine", ["cudf"])
+def test_parquet_reader_empty_pandas_dataframe(tmpdir, engine):
+    df = pd.DataFrame()
+
+    fname = tmpdir.join("test_pq_reader_empty_pandas_dataframe.parquet")
+
+    df.to_parquet(fname)
+    assert os.path.exists(fname)
+
+    expect = pd.read_parquet(fname)
+    got = cudf.read_parquet(fname, engine="cudf")
```

Comment:
Should use the parameter if we're passing it, otherwise we shouldn't parameterize `engine`:
```suggestion
    got = cudf.read_parquet(fname, engine=engine)
```